sdk/resourcemanager/azure-resourcemanager-storage/CHANGELOG.md

```diff
@@ -2,14 +2,10 @@
 
 ## 2.50.0-beta.1 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
 ### Other Changes
 
+- Add `Accepted` to Provisioning State.
```

Comment:
```suggestion
- Added `Accepted` to `ProvisioningState`.
```

Reply:
Fixed by the [commit#e480e85](https://github.com/Azure/azure-sdk-for-java/pull/45027/commits/e480e85086b1e00cbf2e6d8871b4706a4b75c578)